dist2src/core.py

```diff
@@ -313,15 +344,19 @@ def convert(self, origin_branch: str, dest_branch: str):
         Convert a dist-git repository into a source-git repo.
         """
         self.dist_git.checkout(branch=origin_branch)
-        self.source_git.checkout(branch=dest_branch, orphan=True)
+        if not self.source_git.repo.active_branch.name == dest_branch:
```

Comment:
```suggestion
        if self.source_git.repo.active_branch.name != dest_branch:
```

I don't understand why this is needed here, though :thinking:

Reply:
> I don't understand why this is needed here, though thinking

- The `not`?
  - Result of some playing with code probably.
- The conditional?
  - We need to support both `update` (=branch exists) and `create` (=branch does not exist) so we can share as much code as possible.
  - I'll try to improve this a bit.